.github/actions/parse-version/script.php

```diff
@@ -65,25 +65,45 @@ function output(string $name, string $value) {
     $re = '/MOODLE_(.*)_STABLE/m';
     $subst = '$1';
     $coreVersion = preg_replace($re, $subst, $entry['moodle-branch']);
-    $disable_master = !empty($_SERVER['disable_master']);
+    $disable_main = !empty($_SERVER['disable_master']) || !empty($_SERVER['disable_main']);
```

Comment:
Should update README as well

Reply:
Current options haven't changed. `disable_main` should be used going forward but less interested in changing the available options now. Would prefer to have a deprecation warning first and that can be in a new ticket. Mainly just prepping the internal code for it now since I was fixing up an existing 'main' vs 'master' branch check anyways.